### PR TITLE
Create .deb package for ubuntu-18.04 on release

### PR DESCRIPTION
**Pull request by hannes-steffenhagen-diffblue**
Also makes sure to include the name of the distribution in the .deb
packages to make it easier to see which is which.

- [ ] Each commit message has a non-empty body, explaining why the change was made.
- [ ] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [ ] My PR is restricted to a single feature or bugfix.
- [ ] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Review by NlightNFotis — APPROVED** [review]
Looks good to me.

File: .github/workflows/release-packages.yaml [hunk 1]
```diff
@@ -44,6 +44,47 @@ jobs:
...
+        uses: bruceadams/get-release@v1.2.0
+      - name: Upload binary packages
+        uses: actions/upload-release-asset@v1
+        with:
+          upload_url: ${{ steps.get_release_info.outputs.upload_url }}
+          asset_path: ${{ steps.create_packages.outputs.deb_package }}
+          asset_name: ${{ steps.create_packages.outputs.deb_package_name }}
+          asset_content_type: application/x-deb
+
+
```
Comment (chrisr-diffblue) [inline .github/workflows/release-packages.yaml:87]: Seems a bit of a shame that so much of this is a duplication of the the ubuntu-20_04 target - is there any way to re-use this (I don't know github actions, sorry! - but was thinking about either some way of 'including' comment parts, or moving those out into a script/cmake action?)

**Review by chrisr-diffblue — APPROVED** [review]
Looks good to me, and happy for it to go in as is - just a minor query about duplication.

File: .github/workflows/release-packages.yaml (hunk 1 above)
Comment (hannes-steffenhagen-diffblue) [reply .github/workflows/release-packages.yaml:87]: @chrisr-diffblue There is, technically. Although to be honest I've been deliberately avoiding it, I find these short _enough_ and I find logic gets harder to follow as you split it over multiple scripts. This isn't enough script for this to really save us any work, so the main advantage would be it's easier to see that these are doing mostly the same thing – at the cost of hiding what that thing is behind possibly multiple files.

**Pull request merged**